### PR TITLE
LPS-163270 Handle Info Panel scroll

### PR DESCRIPTION
**Pull request by beltranrengifo**
[Jira issue](https://issues.liferay.com/browse/LPS-163270)

## Motivation

When the height of the content inside the sidebar info panel is enough to generate scroll, all the content is displaced.
The user loses the actions button (close, subscribe) and also loses the title, etc.
Now that we are adding tabs along with the assets versions, etc. seems a good moment to modify the scroll behaviour

## Solution proposed

The Sidebar Header is now sticky positioned.
All items that need to be fixed in scrolling are placed inside the Sidebar Header.

## How to verify
- Enable FF `Liferay.FeatureFlags['LPS-161013']`
- Create a web content with enough content to make it scrollable
- Visit the Content Dashboard
- Open the info panel for the web content

**Expected:** The details and versions tabs content should scrollable, not the whole sidebar

image
image

### CODE REVIEW

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/Sidebar.js [hunk 1]
```diff
@@ -28,11 +28,11 @@ const SidebarBody = ({children, className}) => {
 	);
 };
 
-const SidebarHeader = ({children, title}) => {
+const SidebarHeader = ({actionsSlot, children, className, title}) => {
 	const {onClose} = useContext(SidebarContext);
 
 	return (
-		<section className="sidebar-header">
+		<section className={classNames('is-sticky sidebar-header', className)}>
```
Comment (claraizquierdo) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/Sidebar.js:35]: Hi @beltranrengifo , what do you think about applying the is-sticky class only if we have the FF enabled?

**Review by claraizquierdo — APPROVED** [review]

**Comment by beltranrengifo** [comment]
ci:test:sf

**Comment by jkappler** [comment]
ci:test:sf

**Comment by jkappler** [comment]
@beltranrengifo, @claraizquierdo, took an ant all to pick the changes

**Comment by beltranrengifo** [comment]
@jkappler a `ge deploy` commando generated [this changes](https://github.com/liferay-tango/liferay-portal/pull/2836/commits/3d1ea30fc47d98c53e7de8b73d9dc770c4970882#diff-d647fd4ccc0898782576ed62fa0aa30723a79ca92dfd2ebba46128dace09d4f1)
The an `ant all` seems to reverted them
I get confused here 🤣

**Comment by beltranrengifo** [comment]
ci:test:relevant

**Comment by beltranrengifo** [comment]
ci:forward

**Pull request closed without merging**